addons/context_actions/functions/fnc_teleportPlayers.sqf

```diff
@@ -23,7 +23,7 @@ private _players = _this select {isPlayer _x};
     if (_successful) then {
         curatorMouseOver params ["_type", "_entity"];
 
-        if (_type isEqualTo "OBJECT" && {_entity isKindOf "AllVehicles"} && {!(_entity isKindOf "CAManBase")}) then {
+        if (_type == "OBJECT" && {_entity isKindOf "AllVehicles"} && {!(_entity isKindOf "CAManBase")}) then {
```

Comment:
Why remove the `isEqualTo`?

Reply:
Either works here, changed to match other `curatorMouseOver`'s type checks.